sdk-core/transaction.md

```diff
@@ -0,0 +1,46 @@
+## Transaction
+
+```
+class Transaction:
+    constructor({
+        sender: IAddress;
+        receiver: IAddress;
+        gasLimit: IGasLimit;
```

Comment:
To have this in GO (type aliases) we will need to duplicate the FrontEndTransaction defined in mx-chain-core-go which we want to avoid. Why not use regular, default fields?

Reply:
Type aliases would be optional for implementations. `IGasLimit` would be replaced by `uint32` in Go. However, we should have an abstraction (interface) for Address in Go, if possible - whose `String()` returns the bech32 representation.